### ISSUE TITLE
[Cars] locations/signals : Appel de phare

### ISSUE THREAD

**Issue by vosmont** [xee-lab/xee-api-docs#52]
Un appel de phare correspond à la séquence rapide 0;1;0 pour le signal des feux.

Entre l'appel de phare et la disponibilité du signal par l'API Xee, il peut se passer jusqu'à plusieurs minutes et les différents changements sur un signal ne sont pas envoyés (seule la dernière valeur est présente).

Du coup pour détecter un appel de phare via l'API, il faut que le signal des feux soit à 0 avec un timestamp modifié depuis le dernier appel à l'API.

La question est comment détecter un double appel de phare, voire triple ?
Est-il possible d'avoir un nouveau signal pour cet évènement ?

L'évènement double appel de phare peut être utile dans des scénarios liés à la domotique.

**Comment by quentin7b** [comment]
Bonjour Vincent,

Effectivement, **créer un signal** peut être une solution.
Nous allons réfléchir à cela de notre côté.

Néanmoins je ne comprends pas le problème pour l'obtenir de ton côté, si tu utilises l'API `/cars/{carId}/signals` tu peux obtenir les informations sur les feux, et pas seulement la dernière valeur (comme dans le /cars/{carId}/status`.

**Comment by vosmont** [comment]
Effectivement, il faut que je regarde `/cars/{carId}/signals`, ça peut être un moyen pour détecter la séquence.
Ça nécessite de faire des appels supplémentaires à l'API pour qualifier le changement sur le signal `HeadLightSts`.

Par contre le paramètre de début `begin` ne peut pas être la date de dernier appel de l'API, car un signal peut mettre quelques minutes à remonter, mais avec un timestamp correct (ie quelques minutes avant). Il faut donc le faire sur une période flottante.

**Comment by vosmont** [comment]
J'ai testé l'appel à `/cars/{carId}/signals` : pas de trace de `HeadLightSts` (pourtant la date pour ce signal a changé dans `/cars/{carId}/status`)

A noter que l'indicateur `HazardSts` me semble un peu trop présent dans la réponse. Je me demande s'il n'est pas mis à jour à chaque clignotement des feux de détresse, y compris lors d'un changement de direction.
Comme je ne pense pas les utiliser souvent, cela doit correspondre à la fermeture/ouverture de la voiture.

**Comment by quentin7b** [comment]
Effectivement, sur certains véhicules, ces derniers s'activent avec le verrouillage et les clignotants !

En ce qui concerne les `HeadLightSts`, cela est peut-être du au fait qu'ils n'ont pas eu d'update durant la période requêtée ?
